Dockerfile

```diff
@@ -58,7 +58,9 @@ RUN set -x \
     && find /www -type f -exec chmod 640 {} \;
 
 # Add directory for sessions to allow session persistence
-RUN mkdir /sessions
+RUN mkdir /sessions \
+    && mkdir -p /www/tmp \
+    && chmod -R 777 /www/tmp
```

Comment:
Maybe it would be better to have both sessions and twig cache on same directory, to allow persistence for both. But that's something what can be added later.